diff -ruPN cppcheck-1.81/cli/CMakeLists.txt cppcheck-1.81.new/cli/CMakeLists.txt
--- cppcheck-1.81/cli/CMakeLists.txt	2017-11-15 11:56:44.827441212 -0500
+++ cppcheck-1.81.new/cli/CMakeLists.txt	2017-11-15 12:06:04.364405065 -0500
@@ -1,5 +1,10 @@
+find_package(PkgConfig REQUIRED)
+pkg_check_modules (TINYXML2 tinyxml2)
+include_directories(${TINYXML2_INCLUDE_DIRS})
+link_directories(${TINYXML2_LIBRARY_DIRS})
+
+
 include_directories(${PROJECT_SOURCE_DIR}/lib/)
-include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/externals/tinyxml/)
 include_directories(${PROJECT_SOURCE_DIR}/externals/simplecpp/)
 
 file(GLOB hdrs "*.h")
@@ -8,9 +13,9 @@
 list(REMOVE_ITEM srcs ${mainfile})
 
 add_library(cli_objs OBJECT ${hdrs} ${srcs}) 
-add_executable(cppcheck ${hdrs} ${mainfile} $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:tinyxml_objs> $<TARGET_OBJECTS:simplecpp_objs>)
+add_executable(cppcheck ${hdrs} ${mainfile} $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:simplecpp_objs>)
 if (HAVE_RULES)
-    target_link_libraries(cppcheck pcre)
+    target_link_libraries(cppcheck pcre ${TINYXML2_LIBRARIES})
 endif()
 
 install(TARGETS cppcheck
diff -ruPN cppcheck-1.81/CMakeLists.txt cppcheck-1.81.new/CMakeLists.txt
--- cppcheck-1.81/CMakeLists.txt	2017-11-15 11:56:37.575441680 -0500
+++ cppcheck-1.81.new/CMakeLists.txt	2017-11-15 11:58:31.168434342 -0500
@@ -16,7 +16,6 @@
     enable_testing()
 endif()
 
-add_subdirectory(externals/tinyxml)
 add_subdirectory(externals/simplecpp)
 add_subdirectory(lib)   # CppCheck Library 
 add_subdirectory(cli)   # Client application
diff -ruPN cppcheck-1.81/gui/CMakeLists.txt cppcheck-1.81.new/gui/CMakeLists.txt
--- cppcheck-1.81/gui/CMakeLists.txt	2017-11-15 11:48:12.207474328 -0500
+++ cppcheck-1.81.new/gui/CMakeLists.txt	2017-11-15 12:05:49.307406038 -0500
@@ -10,7 +10,11 @@
     endif()
 
     include_directories(${PROJECT_SOURCE_DIR}/lib/)
-    include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/externals/tinyxml/)
+
+    find_package(PkgConfig REQUIRED)
+	pkg_check_modules (TINYXML2 tinyxml2)
+    include_directories(${TINYXML2_INCLUDE_DIRS})
+    link_directories(${TINYXML2_LIBRARY_DIRS})
 
     file(GLOB hdrs "*.h")
     file(GLOB srcs "*.cpp")
@@ -20,9 +24,9 @@
     QT5_ADD_RESOURCES(resources "gui.qrc")
     QT5_ADD_TRANSLATION(qms ${tss})
 
-    add_executable(cppcheck-gui ${hdrs} ${srcs} ${uis_hdrs} ${resources} ${qms} $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:tinyxml_objs> $<TARGET_OBJECTS:simplecpp_objs>)
+    add_executable(cppcheck-gui ${hdrs} ${srcs} ${uis_hdrs} ${resources} ${qms} $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:simplecpp_objs>)
     if (HAVE_RULES)
-        target_link_libraries(cppcheck-gui pcre)
+        target_link_libraries(cppcheck-gui pcre ${TINYXML2_LIBRARIES})
     endif()
     qt5_use_modules(cppcheck-gui ${GUI_QT_COMPONENTS})
 
diff -ruPN cppcheck-1.81/lib/CMakeLists.txt cppcheck-1.81.new/lib/CMakeLists.txt
--- cppcheck-1.81/lib/CMakeLists.txt	2017-11-15 11:56:40.792441473 -0500
+++ cppcheck-1.81.new/lib/CMakeLists.txt	2017-11-15 12:05:31.472407190 -0500
@@ -1,4 +1,8 @@
-include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/externals/tinyxml/)
+find_package(PkgConfig REQUIRED)
+pkg_check_modules (TINYXML2 tinyxml2)
+include_directories(${TINYXML2_INCLUDE_DIRS})
+link_directories(${TINYXML2_LIBRARY_DIRS})
+
 include_directories(${PROJECT_SOURCE_DIR}/externals/simplecpp/)
 
 file(GLOB_RECURSE hdrs "*.h")
diff -ruPN cppcheck-1.81/test/CMakeLists.txt cppcheck-1.81.new/test/CMakeLists.txt
--- cppcheck-1.81/test/CMakeLists.txt	2017-11-15 11:48:12.207474328 -0500
+++ cppcheck-1.81.new/test/CMakeLists.txt	2017-11-15 12:05:55.765405621 -0500
@@ -1,15 +1,20 @@
 if (BUILD_TESTS)
 
+    find_package(PkgConfig REQUIRED)
+	pkg_check_modules (TINYXML2 tinyxml2)
+    include_directories(${TINYXML2_INCLUDE_DIRS})
+    link_directories(${TINYXML2_LIBRARY_DIRS})
+
+
     include_directories(${PROJECT_SOURCE_DIR}/lib/ ${PROJECT_SOURCE_DIR}/cli/)
-    include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/externals/tinyxml)
     include_directories(${PROJECT_SOURCE_DIR}/externals/simplecpp/)
 
     file(GLOB hdrs "*.h")
     file(GLOB srcs "*.cpp")
 
-    add_executable(testrunner ${hdrs} ${srcs} $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:tinyxml_objs> $<TARGET_OBJECTS:simplecpp_objs>)
+    add_executable(testrunner ${hdrs} ${srcs} $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:simplecpp_objs>)
     if (HAVE_RULES)
-        target_link_libraries(testrunner pcre)
+        target_link_libraries(testrunner pcre ${TINYXML2_LIBRARIES})
     endif()
 
     add_custom_target(copy_cfg ALL
